rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/BakeManifestService.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 Netflix, Inc.
```

Comment:
License header should be `Armory` instead of `Netflix, Inc` for all new files.

Reply:
changed